k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -1120,13 +958,11 @@ function deployCortxServer()
 
     printf "\nWait for CORTX Server to be ready"
     # Wait for all cortx-data deployments to be ready
-    declare -a deployments
+    local deployments=()
     for i in "${!node_selector_list[@]}"; do
-        node_name=${node_name_list[i]}
-        deployments[${#deployments[@]}]="deployment/cortx-server-${node_name}"
```

Comment:
This is an "unbound" variable in bash, which was causing problems for me. `declare -a` makes an array, not an associative array (which is `declare -A`), and since this is just an array, I changed the usage to that.